system-int-test/test/cmr/system_int_test/search/collection_keyword_search_test.clj

```diff
@@ -205,18 +209,37 @@
 
         ;; entry title
         "coll1" [coll1]
+        "\"coll1\"" [coll1]
+        "coll26" [coll26 coll26-1]
+         "\"coll26\"" [coll26 coll26-1]
         "Mitch made a (merry-go-round)" [coll2]
+        "\"Mitch made a (merry-go-round)\"" [coll2]
         "(merry-go-round)" [coll2]
+        "\"(merry-go-round)\"" [coll2]
         "merry-go-round" [coll2]
+        "\"merry-go-round\"" [coll2]
         "merry" [coll2]
+        "\"merry\"" [coll2]
         "merry go round" [coll2]
+        "\"merry go round\"" []
         "merry-go" []
+        "\"merry-go\"" []
+
+        ;; mix of keyword and keyword phrase search: not supported yet.
+        "Mitch \"a (merry-go-round)\"" []
```

Comment:
I think we should return a validation error on the un-supported parameter combinations rather than silently return nothing. Let's add a validation rule for keyword search parameters and handle these two cases.